extensions/integration-tests/src/objectExplorer.test.ts

```diff
@@ -11,63 +11,86 @@ import { context } from './testContext';
 import { getBdcServer, TestServerProfile, getAzureServer, getStandaloneServer } from './testConfig';
 import { connectToServer } from './utils';
 import assert = require('assert');
+import { stressify } from '../testfmks/src/stress';
 
 if (context.RunTest) {
 	suite('Object Explorer integration suite', () => {
 		test('BDC instance node label test', async function () {
-			const expectedNodeLabel = ['Databases', 'Security', 'Server Objects', 'Data Services'];
-			let server = await getBdcServer();
-			await VerifyOeNode(server, 6000, expectedNodeLabel);
+			await (new ObjectExplorerTester()).bdcNodeLabelTest();
 		});
 		test('Standard alone instance node label test', async function () {
-			if (process.platform === 'win32') {
-				const expectedNodeLabel = ['Databases', 'Security', 'Server Objects'];
-				let server = await getStandaloneServer();
-				await VerifyOeNode(server, 3000, expectedNodeLabel);
-			}
+			await (new ObjectExplorerTester()).standaloneNodeLabelTest();
 		});
 		test('Azure SQL DB instance node label test', async function () {
-			const expectedNodeLabel = ['Databases', 'Security'];
-			let server = await getAzureServer();
-			await VerifyOeNode(server, 3000, expectedNodeLabel);
+			await (new ObjectExplorerTester()).azureDbNodeLabelTest();
 		});
 		test('context menu test', async function () {
-			let server = await getAzureServer();
-			await connectToServer(server, 3000);
-			let nodes = <azdata.objectexplorer.ObjectExplorerNode[]>await azdata.objectexplorer.getActiveConnectionNodes();
-			assert(nodes.length > 0, `Expecting at least one active connection, actual: ${nodes.length}`);
+			await (new ObjectExplorerTester()).contextMenuTest();
+		});
+	});
+}
 
-			let index = nodes.findIndex(node => node.nodePath.includes(server.serverName));
-			assert(index !== -1, `Failed to find server: "${server.serverName}" in OE tree`);
+class ObjectExplorerTester {
+	private static IterationCount = 20;
+	private static ParallelCount = 1;
```

Comment:
this is a nit, but i'd like our tests to be consistent on how they pass in iterations/dop. In one class we hardcode the numbers in the stressify decorator; here, we put the variables above. Total nit, I know, but just for consistency.

Reply:
we have these variables in Notebook.test.ts as well. So currently there is no hardcoding in @stressify calls. If every method would require different stress parameter then i guess it would be overkill to add a variable for each value. But in case you want to have a class wide policy different from the default policy in Stress framkeworks then use these static variables.

Reply:
These can just be const. I'd also suggesting just naming them dop and iterations, but that's just a preference nit on my part.

Although TBH I don't see what's wrong with just having the numbers in the decorators themselves. It's annoying to have to scroll up just to see the actual values and I don't see these changing much (if ever). I think Chris was just saying be consistent in whether we use hardcoded numbers or consts which I agree with - so if you/others think it's best to have defined values for these then that's fine by me. 

---
In reply to: [284937816](https://github.com/microsoft/azuredatastudio/pull/5499#discussion_r284937816) [](ancestors = 284937816)